### PR TITLE
Upgraded Soundflower to 2.0b2 and new maintainer.

### PR DESCRIPTION
**Pull request by ohammersmith**
This version is signed so does not require enabling unsigned kexts on El Capitan.

See https://github.com/RogueAmoeba/Soundflower-Original/issues/39 and https://github.com/mattingalls/Soundflower/releases and https://rogueamoeba.com/freebies/soundflower/

### CODE REVIEW

**Comment by adityadalal924** [comment]
Is `:allow_untrusted => true` still needed, then?

**Comment by ohammersmith** [comment]
The `:allow_untrusted => true` is still required. I just tested without it. It must be that the .pkg itself isn't signed. I'll see if he can sign the pkg.

```
17:29 $ brew cask install soundflower
==> Downloading https://soundflower.googlecode.com/files/Soundflower-1.6.6b.dmg
######################################################################## 100.0%
==> Running installer for soundflower; your password may be necessary.
==> Package installers may write to any location; options such as --appdir are i
Password:
==> installer: Package name is Soundflower
==> installer: Certificate used to sign package is not trusted. Use -allowUntrus
Error: Command failed to execute!

==> Failed command:
["/usr/bin/sudo", "-E", "--", "/usr/sbin/installer", "-pkg", "#<Pathname:/opt/homebrew-cask/Caskroom/soundflower/1.6.6b/Soundflower.pkg>", "-target", "/"]

==> Output of failed command:
installer: Package name is Soundflower
installer: Certificate used to sign package is not trusted. Use -allowUntrusted to override.


==> Exit status of failed command:
#<Process::Status: pid 17259 exit 1>
```

I'm taking a look at the uninstall now.

**Comment by ohammersmith** [comment]
Scratch that, `:allow_untrusted => true` is _not_ required. I had switched machines and forgot to grab my branch of Homebrew Cask.

**Comment by ohammersmith** [comment]
> Also, I see there is a Uninstall.scpt with the following, would you mind adding the contents of that to the uninstall stanza? If you need any help doing so, just ask!
> 
> ```
>     do shell script "sudo rm -rf /System/Library/Extensions/Soundflower.kext" with administrator privileges
>     do shell script "sudo rm -rf /Library/Extensions/Soundflower.kext" with administrator privileges
>     do shell script "sudo rm -rf /Library/Receipts/Soundflower*" with administrator privileges
>     do shell script "sudo rm -rf /var/db/receipts/com.cycling74.soundflower.*" with administrator privileges
>     do shell script "sudo rm -rf /Applications/Soundflower" with administrator privileges
> ```

None of those are actually required after running `brew cask uninstall soundflower`. I suspect most if it is meant to nuke any possible conflict with older versions.

The existing `uninstall` stanza takes care of `/Applications/Soundflower` and the kext in `/Library/Extensions`.

I'll update the PR without the `:allow_untrusted => true`.

File: Casks/soundflower.rb [hunk 1]
```diff
@@ -1,13 +1,14 @@
 cask :v1 => 'soundflower' do
-  version '1.6.6b'
-  sha256 '61ca31d7478d057e32caaeac3c739f965ba9eb2a27570b3cc715e706d4740dfb'
+  version '2.0b2'
+  sha256 '6b5e56d53238cf0f9075886aa40580634fc9d23368239f22eccebfd97c9f8e34'
+
+  url "https://github.com/mattingalls/Soundflower/releases/download/#{version}/Soundflower-#{version}.dmg"
```
Comment (jawshooah) [inline Casks/soundflower.rb]: Please delete this empty line.

File: Casks/soundflower.rb [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 
-  url "https://soundflower.googlecode.com/files/Soundflower-#{version}.dmg"
   name 'Soundflower'
-  homepage 'https://code.google.com/p/soundflower/'
+  homepage 'https://github.com/mattingalls/Soundflower'
   license :oss
 
-  pkg 'Soundflower.pkg', :allow_untrusted => true
+  pkg 'Soundflower.pkg'
```
Comment (jawshooah) [inline Casks/soundflower.rb]: Please add a newline between the `pkg` and `postflight` stanzas.

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 1124f8db3374385fead3c11f7fd4984d6235c8d4. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**